Source/Evolve/WarpXEvolve.cpp

```diff
@@ -385,7 +385,7 @@ WarpX::OneStep_nosub (Real cur_time)
         EvolveF(0.5*dt[0], DtType::SecondHalf);
 
         EvolveB(0.5*dt[0]); // We now have B^{n+1}
-
+        
         //why not implementing FillBoundary here? possibly: implemented in if{safe_guard_cells} Line 452
```

Comment:
make sure there are no white spaces.  once these white spaces are gone, this file won't even show up in the PR as having any changes!